app/controllers/concerns/authorize_resource.rb

```diff
@@ -21,7 +21,13 @@ def authorize_many(resources, privilege)
 
   def auth(user, privilege, resource)
     unless user.allowed_to?(privilege, resource)
-      logger.info "Current user '#{user.role_id}' is not permitted to '#{privilege}' resource '#{resource.resource_id}'"
+      logger.info(
```

Comment:
Why not using logger.error instead of info?

Reply:
According to Datadog's post about [managing Rails application logs](https://www.datadoghq.com/blog/managing-rails-application-logs/), error logs indicate `errors that cause an operation to fail (e.g., missing data or files) but not the application`. In this case the operation didn't fail and we got to the conclusion successfully. We want to notify the client that there was an error in his flow so it's in info